jumpstart-week2/README.md

```diff
@@ -0,0 +1,222 @@
+
+## HTML
+
+HTML is a subset of XML.
+
+#### What is XML?
+
+XML defines a syntax (a set of rules) to express data. There are 3 main aspects of XML.
+
+A tag (or element) is the basic unit.
+
+Example: `<element>`
+
+Every tag has a name which is the first word following immediately after the opening bracket (`<`). Every tag also has to have a closing tag after it. A closing tag is simply `</element>` where `element` is just the name of the tag repeated again. 
+
+Between the opening and closing tags are the second aspect of XML, children. Between an element's two tags is a list of either other XML tags or text. Every item in this list is a child of the element that is surrounding them. Of course, an XML element that is the child of an element can also have its own children. You'll often here the term "depth" of an XML element to refer to the number of parents that element has.
+
+```xml
+<parent>
+    <first-child>
+        Richard
+        <first-grandchild>
+            Jamie
+        </first-grandchild>
+        <second-grandchild>
+            Jessica
+        </second-grandchild>
+    </first-child>
+    <second-child>
+        John
+    </second-child>
+    <third-child>
+        Dylan
+        <third-grandchild>
+          Alice
+        </third-grandchild>
+    </third-child>
+</parent>
+```
+
+> Side note: Often it'll be the case that an XML element has zero children. You can still write out both the opening and closing tags with nothing between them but XML also has a shorthand that saves some time retyping the closing tag:
+> `<element></element` === `<element />`
+
+The third aspect of XML is attributes. Attributes live within the opening tag of XML elements. Attributes have a name and a value, both of which are just plain strings (words). The attribute value will always be wrapped in double quotes or single quotes.
+
+`<element attributeName="attributeValue">`
+
+Attributes allow you to specify extra information about an element.
+
+#### How is HTML different than XML?
+
+Like we said earlier, HTML is a subset of XML. All HTML is also XML, not all XML is HTML.
+
+HTML defines several XML elements and gives guidelines on how they can be used together. There are a ton of different HTML elements that are defined and a lot of them aren't super useful so don't bother trying to learn all of them (google is your friend though).
+
+A couple of the important HTML elements to know are:
+
+- `<div>`: Defines a block element
+- `<span>`: Defines an inline element
+- `<p>`: Stands for paragraph, use this to wrap text
```

Comment:
Maybe add `link` and `title` here too!

Reply:
Should we also talk about `head`, `body`, `<!DOCTYPE html>`, and `html` special html elements?

Reply:
Adding on to this, I think it could be helpful to briefly define "inline" and "block," otherwise "defines a block element" won't mean much to people who aren't already familiar with HTML. Or we can describe divs/spans another way.